lib/json_schema_form/umm_form_elements/umm_text_field.rb

```diff
@@ -1,10 +1,10 @@
 # :nodoc:
 class UmmTextField < UmmFormElement
   def render_markup
-    text_field_tag(keyify_property_name(form_fragment), text_value, element_properties(schema_fragment))
+    text_field_tag(keyify_property_name, text_value, element_properties(schema_fragment))
   end
 
   def render_preview
-    content_tag(:p, element_value)
+    content_tag(:p, text_value)
```

Comment:
this change was just updating render_preview to use the same value as render_markup, a change from 1014